### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by angelacuahutle**
Dear TSE, 

For this Milestone, I focused on the look and feel of this project. I have implemented the following elements :

-Now you should improve the look and feel of the application, adding the styles of your choice.

-Please keep the general layout of the wireframe, this is the only mandatory requirement.

-Use plain CSS framework.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @angelacuahutle 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+  box-shadow: 4px 3px black;
+  background: white;
+  cursor: pointer;
+  user-select: none;
+}
+
+* {
+  /* stylelint-disable */
+  font-family: 'Balsamiq Sans';
+  /* stylelint-enable */
```
Comment (mikenath223) [inline src/style.css:24]: - [x] Please remove this stylelint disable line. Its best to fix the issue in the code.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Leaderboard
 In this project I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. In the first stages I will be working on the version of the app following a wireframe, but without styling - just focus on functionality. In following activities, I will consume the Leaderboard API using JavaScript async and await and add some styling.
 
-![screenshot](./milestone2.png)
+![screenshot](./final-style.png)
```
Comment (mikenath223) [inline README.md:6]: - [x] Kindly deploy the project online and provide the right link on the readme.

**Comment by angelacuahutle** [comment]
Implemented changes!!

**Comment by Davidosky007** [comment]
## Hi @angelacuahutle 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
done-gif



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**